application/views/admin/export/generatestats/_header.php

```diff
@@ -30,7 +30,8 @@
         <?php echo CHtml::form(["responses/browse/", 'surveyId' => $surveyid], 'post') . "\n"; ?>
         <p>
             <?php Yii::app()->user->setState('sql_' . $surveyid, $sql); ?>
-            <input type='submit' class='btn btn-outline-secondary d-print-none' value='<?php eT("Browse"); ?>'/>
+            <input type='submit' class='btn btn-outline-secondary d-print-none statistics-browse' value='<?php eT("Browse"); ?>'
+                   data-browseurl='<?php echo App()->createUrl('responses/browse/', ['surveyId' => $surveyid]) ?>'/>
```

Comment:
the small form cant be rendered inside another form this is why this fails to send the form should be removed and just a link with the url wil be enough to fix it